packages/imperative/src/rest/src/session/ConnectionPropsForSessCfg.ts

```diff
@@ -161,20 +172,31 @@ export class ConnectionPropsForSessCfg {
         this.loadSecureSessCfgProps(connOptsToUse.parms, promptForValues);
 
         if (connOptsToUse.getValuesBack == null && connOptsToUse.doPrompting) {
-            connOptsToUse.getValuesBack = this.getValuesBack(connOptsToUse);
+            connOptsToUse.getValuesBack = this.getValuesBack(connOptsToUse as any);
```

Comment:
We might be able to remove the `as any` here by updating the method definition of `getValuesBack` to use a type generic like the following:
```typescript
private static getValuesBack<SessCfgType extends ISession=ISession>(connOpts: IOptionsForAddConnProps<SessCfgType>):
```